var/spack/repos/builtin/packages/skit/package.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Skit(MakefilePackage):
```

Comment:
The project's name is `SPARSKIT`, even though the library it produces is `libskit`. The spack package name should be the project name if possible.

Reply:
OK, I rename the package.